chainer/links/connection/convolution_nd.py

```diff
@@ -22,14 +22,14 @@ class ConvolutionND(link.Link):
         pad (int or tuple of ints): Spatial padding width for input arrays.
             ``pad=p`` and ``pad=(p, p, ..., p)`` are equivalent.
         nobias (bool): If ``True``, then this function does not use the bias.
-        initialW (array): Initial weight array. If ``None``, the default
-            initializer is used.
+        initialW (None, callable, numpy.ndarray, or cupy.ndarray):
```

Comment:
* It does not accept ``cupy.ndarray``
* It accepts scalar and `Initializer`.

Reply:
Note that `Initializer` is callable. Should we add the explanation that it can accept `Initializer` in particular?

Reply:
I think so. `Initializer` is particularly designed for this use, thus users should be notified of it's existence.

Reply:
OK, should we merge #3327 first?

Reply:
Ah, that sounds good!

Reply:
I changed the status label to `blocked-by-another-pr`.